.travis.yml

```diff
@@ -9,16 +9,10 @@ env:
   global:
     - ROS_REPO=ros
   matrix:
-    - ROS_DISTRO=indigo UPSTREAM_WORKSPACE=file
-    - ROS_DISTRO=indigo UPSTREAM_WORKSPACE=debian
-    - ROS_DISTRO=kinetic UPSTREAM_WORKSPACE=file
-    - ROS_DISTRO=kinetic UPSTREAM_WORKSPACE=debian
-matrix:
-  allow_failures:
-    - env: ROS_DISTRO=indigo UPSTREAM_WORKSPACE=debian
-    - env: ROS_DISTRO=kinetic UPSTREAM_WORKSPACE=debian
+    - ROS_DISTRO=melodic UPSTREAM_WORKSPACE=file
+    - ROS_DISTRO=melodic UPSTREAM_WORKSPACE=debian
```

Comment:
if we merge this into a `kinetic_dev` branch (kinetic/melodic) we should include the kinetic jobs again and provide respective rosinstall file (if needed)